src/main/scala/com/linkedin/feathr/offline/source/pathutil/LocalPathChecker.scala

```diff
@@ -20,13 +21,33 @@ private[offline] class LocalPathChecker(hadoopConf: Configuration) extends PathC
     LocalFeatureJoinUtils.getMockPathIfExist(path, hadoopConf, None).isDefined
   }
 
+  /**
+   * check whether the input path is an Hdfs path. Neesd to have separate function, as there are class conflicts with Breaks.getClass
+   * @param path input path.
+   * @return true if the path is an Hdfs path.
+   */
+  def isHdfs(path: String): Boolean = {
```

Comment:
From the implementation, it is not related to HDFS, but whether it has handler or not. Change to a better name?

Reply:
sg, changed it isExternalDataSource.